src/test/java/edu/harvard/iq/dataverse/util/PersonOrOrgUtilTest.java

```diff
@@ -89,6 +89,7 @@ public void testName() {
             verifyIsPerson("kcjim11, kcjim11", "kcjim11", "kcjim11");
             
             verifyIsPerson("Bartholomew 3, James", "James", "Bartholomew 3");
+            verifyIsPerson("Smith, ", null, "Smith");
```

Comment:
What happens if there is no space at the end? Should you add that as another test case?

Reply:
Verifying failed because the compared JsonObject was normalized via an util method. I added the normalization to the comparison in the assertion and added the other test case.